src/main/resources/patch_2_mysql.sql

```diff
@@ -1 +1 @@
-ALTER TABLE islands MODIFY name VARCHAR(512) NULL
\ No newline at end of file
+ALTER TABLE islands MODIFY name VARCHAR(512) NULL
```

Comment:
Whats this?

Reply:
I did it, just added a newline because they are important